app/routes/pricing.tsx

```diff
@@ -140,9 +140,12 @@ const pricingData: PricingData = {
         components: [
           { name: "Requests", included: "None", unit: "per million", price: "$0.15" },
           { name: "Duration", included: "None", unit: "per million GB-s", price: "$12.50" },
-          { name: "Rows Read", included: "None", unit: "per million rows", price: "$0.0001" },
-          { name: "Rows written", included: "None", unit: "per million rows", price: "$1.00" },
+          { name: "SQL Rows Read", included: "None", unit: "per million rows", price: "$0.001" },
+          { name: "SQL Rows Written", included: "None", unit: "per million rows", price: "$1.00" },
           { name: "SQL Stored data", included: "None", unit: "per GB-month", price: "$0.20" },
+          { name: "KV Reads", included: "None", unit: "per million rows", price: "$0.20" },
```

Comment:
For last 4 rows to match wording on https://developers.cloudflare.com/durable-objects/platform/pricing/:

```
Read request units (KV storage backend), per million 4KB units of data read
Write request units (KV storage backend), per million 4KB units of data written
Delete requests (KV storage backend), per million requests 
Stored data (KV storage backend), per GB-month
```